app/tpl/index.hbs

```diff
@@ -3,16 +3,26 @@
         <header class="gh-header">
             <h1>Scan Your Theme</h1>
             <p class="gh-subhead">Upload a zip to check for errors, deprecations and other compatibility issues.</p>
-            <p class="gh-subhead warning"><strong>GScan validates your theme based on the Ghost 1.0 requirements.</strong></p>
-            <a href="https://themes.ghost.org/v1.25.0/docs/changelog#ghost-100" target=_blank class="gh-morelink">
-                See the changelog for Ghost 1.0 themes
+
+            <a href="https://themes.ghost.org/v2.0.0/docs/changelog" target=_blank class="gh-morelink">
```

Comment:
If we release another minor v2, we would need to change these links again.
I would point them to latest: https://themes.ghost.org/.

Reply:
Yes, I am aware of that. Still thinking of a better solution for that...